site/docs/reference/Connectors/capture-connectors/gcs.md

```diff
@@ -73,4 +73,35 @@ captures:
 
 Your capture definition may be more complex, with additional bindings for different GCS prefixes within the same bucket.
 
-[Learn more about capture definitions.](../../../concepts/captures.md#pull-captures)
\ No newline at end of file
+[Learn more about capture definitions.](../../../concepts/captures.md#pull-captures)
+
+### Configure Google service account impersonation
```

Comment:
Maybe change the heading to something like "Advanced Usage: Service Account Impersonation" to make it clearer to the casual reader that this isn't a normal/necessary part of using the GCS connector?